### PR TITLE
Add JSON to YAML converter

### LINKED ISSUE #330: [Proposal] json to yaml convertor
# Summary

Create a json to yaml convertor

# Example search queries

- json to yaml

# Implementation (optional)

You can create `json to yaml` by following the implementation of [json to csv](https://github.com/felvin-search/instant-apps/tree/7538bef499fafac0ef55e94b4fa4f7032a5f1732/apps/json-to-csv) and 
[yaml to json](https://github.com/felvin-search/instant-apps/tree/master/apps/yaml-to-json) and [json to csv] apps.

# Context

### PR DESCRIPTION
**Pull request by Sync271**
Closes #330 
https://www.loom.com/share/d44e4153ef4d4ae9828a163fc98ed7bf

### CODE REVIEW

File: apps/json-to-yaml/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  const [copy, setCopy] = useState(false);
+  const [yamlData, setYAMLData] = useState();
+
+  useEffect(() => {
+    if (jsonData) {
+      try {
+        const yamlDoc = yaml.dump(JSON.parse(jsonData));
+        setYAMLData(yamlDoc);
+      } catch (e) {
+        console.log("Invalid JSON");
```
Comment (hargup) [inline apps/json-to-yaml/src/App.jsx:135]: Ideally we should show the invalid JSON error on the screen to the user.
The space below the text box should be a good place to put this error.

**Review by hargup — APPROVED** [review]
LGTM except for the comment on LN135, @Sync271 thanks for your work 😄

**Comment by hargup** [comment]
The little loom very clearly shows how the app works, thanks for that as well.

File: apps/json-to-yaml/src/App.jsx (hunk 1 above)
Comment (Sync271) [reply apps/json-to-yaml/src/App.jsx]: I've added a span that is displayed below the text box if the JSON data is invalid

**Comment by hargup** [comment]
Works well
Screenshot 2022-03-12 at 2 21 55 PM

**Pull request merged**